### PR TITLE
Make monitor module use external volume always

### PR DESCRIPTION
**Pull request by ymorimo**
https://scalar-labs.atlassian.net/browse/DLT-6177

Removed the option `enable_log_volume` from the monitor module. An external volume should always be created to store logs.

### CODE REVIEW

File: modules/azure/monitor/main.tf [hunk 1]
```diff
@@ -26,7 +26,7 @@ module "monitor_cluster" {
 }
 
 resource "azurerm_managed_disk" "monitor_log_volume" {
-  count = local.monitor.enable_log_volume ? local.monitor.resource_count : 0
+  count = local.monitor.resource_count
```
Comment (feeblefakie) [inline modules/azure/monitor/main.tf:29]: Not checking if `enable_tdagent` is a bug of the Azure module?

**Review by feeblefakie — COMMENTED** [review]
Thanks! Left 1 question.

File: modules/azure/monitor/main.tf (hunk 1 above)
Comment (ymorimo) [reply modules/azure/monitor/main.tf]: Ah, right. Updated to check the variable for the Azure module to match the AWS in this PR: 1c86ddb

**Review by tei-k — APPROVED** [review]
LGTM! 👍

**Review by feeblefakie — APPROVED** [review]
LGTM!

**Pull request merged**